src/components/timeRangePicker/components/DateHourPicker/components/TimeRangeLabel.tsx

```diff
@@ -0,0 +1,22 @@
+import { TimeRangeOption } from "../../../types";
+import { FlexContainer } from "../../styles";
+import { CountTag } from "./styles";
+
+type Props = {
+  timeRangeOption: TimeRangeOption;
+  hasCounts: boolean;
+};
+
+export function TimeRangeLabel({ timeRangeOption, hasCounts }: Props) {
+  return (
+    <FlexContainer
+      justifyContent={hasCounts ? "space-between" : "center"}
+      alignItems="stretch"
+    >
+      <label>{`${timeRangeOption.start} - ${timeRangeOption.end}`}</label>
+      {![undefined, 0].includes(timeRangeOption.count) && (
```

Comment:
`undefined` is default, right? Maybe we could do `0` as default to keep type consistent